.changeset/chatty-ravens-collect.md

```diff
@@ -0,0 +1,12 @@
+---
+"@vivjs/constants": minor
+"@vivjs/extensions": minor
+"@vivjs/layers": minor
+"@vivjs/loaders": minor
+"@hms-dbmi/viv": minor
+"@vivjs/types": minor
+"@vivjs/viewers": minor
+"@vivjs/views": minor
+---
+
+(chore): upgrade deck.gl + luma.gl
```

Comment:
One nit. I'd but more details here about specifically what changed (i.e., what versions, that this is breaking). With changesets you have the space to explain why the changes are involved. This is how we communicate to developers what something has changed.


For example I always appreciate esbuilds changelogs: https://github.com/evanw/esbuild/blob/main/CHANGELOG.md#0240

They let me know as the user what I can expect.


```suggestion
Upgrade deck.gl to XXXX and Luma.gl to XXXX

This upgrade is **breaking**. You will need to... 
```

Reply:
Great call, added some deatil.